### DISCUSSION TITLE
Add translation memory/glossary

### DISCUSSION THREAD

**Discussion by aardvarkk** [mirego/accent#336] [Ideas]
This is an enhancement request.

We have a ton of similar/duplicated text in our translation project that needs to be consistently translated. Without a fast glossary or memory of previous translations, it's tedious to keep re-translating the same text over and over. A find-and-replace or glossary of terms (which shows suggestions) would be very helpful to speed up the translation process.

**Comment by simonprev** [comment]
Great suggestion, I’ve been thinking on how to implement this in the UI by looking at other products like Crowdin, Phraseapp, Transifex…

Will add this to the backlog 👍
